internal/archive/archive.go

```diff
@@ -285,6 +289,17 @@ func WriteZipToTar(tw *tar.Writer, srcZip, basePath string, uid, gid int, mode i
 	return nil
 }
 
+func overwritePermissions(header zip.FileHeader) bool {
+	var (
+		creatorFAT  uint16 = 0
+		creatorVFAT uint16 = 14
+	)
+
+	// This identifies FAT files, based on the `zip` source: https://golang.org/src/archive/zip/struct.go
+	firstByte := header.CreatorVersion >> 8
+	return firstByte == creatorFAT || firstByte == creatorVFAT
+}
+
```

Comment:
may we could call this function `isFAT` becaause that's what is does?
or `shouldOverwrite`?